components/k8s-model-server/inception-client/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:2.7.14
+
+RUN pip install grpcio tensorflow tensorflow-serving-api
+
+RUN mkdir -p /opt/label
+
+RUN mkdir -p /data
```

Comment:
`VOLUME` should create the directory if the dir is not exist.  Or how about combining line 19 and 21 together?

Reply:
And done - got rid of the `VOLUME` bit